### ISSUE TITLE
ProgressBar witdh from GONE to VISIBLE

### ISSUE THREAD

**Issue by HectorFGarcia** [ZieIony/Carbon#223]
When I change ProgressBar's visibility, the width changes too.

Example

```
<carbon.widget.ProgressBar
    android:layout_gravity="center"
    android:layout_margin="@dimen/carbon_padding"
    android:id="@+id/wait_bar"
    app:carbon_progressStyle="circular_indeterminate"
    app:carbon_tint="@color/white"
    app:carbon_barWidth="5dp"
    android:layout_width="40dp"
    android:layout_height="40dp"/>
```

This works fine.

![pb2](https://cloud.githubusercontent.com/assets/12731574/14144057/856fff1c-f68e-11e5-8ac2-76ad7d04752d.png)

But if I add this to the XML...

```
 android:visibility="gone"
```

And then I do this...

```
waitBar.setVisibility(View.VISIBLE);
```

This is what happens.

![pb1](https://cloud.githubusercontent.com/assets/12731574/14144107/bbb67e20-f68e-11e5-82f8-c6edc089a272.png)

**Comment by HectorFGarcia** [comment]
I fixed it for now wrapping the ProgressBar inside a LinearLayout.

**Comment by ZieIony** [comment]
Did you try the most recent commit? I guess this issue was fixed already

**Comment by ZieIony** [comment]
18 without a comment, I assume that it's working. In a meantime I've released 0.13.0
